### ISSUE TITLE
Feature Request: FMS SEL ALT

### ISSUE THREAD

**Issue by MatisseBE** [Sequal32/vrclivetraffic#18]
Adsbexchange presents the FMS SEL ALT for mode-s aircraft.
We can use this data to set the CFL for aircraft, using a plugin for Euroscope. There's already quite some plugins out there that set an initial CFL if you want some references.

![image](https://user-images.githubusercontent.com/51272243/162569074-9f16244a-5b6a-4a29-b4c4-4b687beeece3.png)

**Comment by simonA359** [comment]
I agree, it would be a great idea, besides adding the current KIAS to replace the assigned speed, for instance.
